catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/gmti/NitfGmtiTransformer.java

```diff
@@ -78,23 +77,28 @@ public Metacard transform(NitfSegmentsFlow nitfSegmentsFlow, Metacard metacard)
     }
 
     private void handleHeader(TaggedRecordExtensionHandler header, Metacard metacard) {
-        List<Tre> tres = header.getTREsRawStructure().getTREs();
-
-        tres.stream().filter(tre -> ACFTB.equals(tre.getName().trim()))
-                .forEach(tre -> handleSegmentHeader(metacard, tre, AcftbAttribute.values()));
-
-        tres.stream().filter(tre -> MTIRPB.equals(tre.getName().trim())).forEach(tre -> {
-            handleSegmentHeader(metacard, tre, MtirpbAttribute.values());
-
-            try {
-                List<TreGroup> targets = tre.getEntry(TARGETS).getGroups();
-
-                targets.stream().forEach(group -> handleSegmentHeader(metacard, group,
-                        IndexedMtirpbAttribute.values()));
-            } catch (NitfFormatException e) {
-                LOGGER.debug("Could not parse NITF target information: ", e);
-            }
-        });
+        List<Tre> tres = header.getTREsRawStructure()
+                .getTREs();
+
+        handleTres(metacard, header);
+
+        tres.stream()
+                .filter(tre -> MTIRPB.equals(tre.getName()
+                        .trim()))
+                .forEach(tre -> {
+                    try {
+                        List<TreGroup> targets = tre.getEntry(TARGETS)
+                                .getGroups();
+
+                        targets.stream()
+                                .forEach(group -> handleSegmentHeader(metacard,
+                                        group,
+                                        IndexedMtirpbAttribute.values()));
+                    } catch (NitfFormatException e) {
+                        LOGGER.warn("Could not parse NITF target information: {} " + e.getMessage(),
```

Comment:
Was this deliberately reverted to the old warn level, or was there a possibility this was merged / rebased incorrectly?

See DDF-2390 -https://github.com/codice/alliance/commit/ddd398490d8e91c682e5718ca87c251aaa4ef376#diff-35230559684c153d7a3432f20c908f3eR95